Auro-rerun -> Auto-retry (#6646)
PipedreamHQ/pipedream: docs/docs/workflows/settings/README.md

<img src="https://res.cloudinary.com/pipedreamin/image/upload/v1656631849/docs/Screen_Shot_2022-06-30_at_4.30.44_PM_oauty4.png" width="200px" alt="Notify me on errors toggle">

> ## Auto-retry Errors

Author: ```suggestion
## Auto-retry errors
```

Author: Changed "Errors" to lowercase "errors" to follow Google's style guide on capitalization.

By default, if a step fails on all 8 retries, and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel. But sometimes you need to investigate errors as soon as they happen. If you're connecting to your database, and receive an error that the DB is down, you may want to investigate that immediately.

> On any workflow with auto-retry enabled, you can optionally choose to **Send notification on first error**. This is disabled by default so you don't get emails for transient errors, but you can enable for critical workflows where you want visibility into all errors.

Author: ```suggestion
On any workflow with auto-retry enabled, you can optionally choose to **Send notification on first error**. This is disabled by default, so you don't receive emails for transient errors. However, you can enable it for critical workflows where you want visibility into all errors.
```

Author: Changed "get" to "receive" and "but" to "however" to improve clarity and readability according to Google's style guide and Dreyer's English.
